UIKitDZ/UIKitDZ/Notifications/Cells/NotificationComment.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NotificationComment.swift
+//  UIKitDZ
+//
+//  Created by Киса Мурлыса on 21.02.2024.
```

Comment:
```suggestion
//  Created by Киса Мурлыса on 21.02.2024.
```
так мырлыса что с форматом где шапка нормальная